From 193c8d8c55a3478ca5c9e161ce581e5794098c6d Mon Sep 17 00:00:00 2001
From: Olivier Schonken <olivier.schonken@gmail.com>
Date: Thu, 21 Jan 2016 23:04:49 +0100
Subject: [PATCH] Do not use genstrings

Using cross compiled genstrings while cross-compiling will break
compilation.  Genstrings is still compiled because some power users
might want to do ppd compilation on their target machine, and for that
they would need to use the native compiled genstrings binary, to
generate a sample.c file for their target

Signed-off-by: Olivier Schonken <olivier.schonken@gmail.com>
[Thomas: completely remove call to genstrings rather than commenting
it, remove useless and potentially confusing 'echo' before the
genstrings call.]
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
[Fabrice: updated for 2.3.0]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Michael: updated for 2.3.3]
Signed-off-by: Michael Trimarchi <michael@amarulasolutions.com>
---
 ppdc/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/ppdc/Makefile b/ppdc/Makefile
index 32e2e0b..7b18879 100644
--- a/ppdc/Makefile
+++ b/ppdc/Makefile
@@ -186,8 +186,6 @@ genstrings:		genstrings.o libcupsppdc.a ../cups/$(LIBCUPSSTATIC) \
 	$(LD_CXX) $(ARCHFLAGS) $(ALL_LDFLAGS) -o genstrings genstrings.o \
 		libcupsppdc.a $(LINKCUPSSTATIC)
 	$(CODE_SIGN) -s "$(CODE_SIGN_IDENTITY)" $@
-	echo Generating localization strings...
-	./genstrings >sample.c
 
 
 #
-- 
2.17.1

